### PR TITLE
Port PCB Assembly doc to markdown

### LINKED ISSUE #222: Port PCB Assembly doc to markdown
The [doc](https://github.com/nasa-jpl/open-source-rover/blob/master/Electrical/PCB%20Assembly.pdf) to be ported

Refer to [the project](https://github.com/nasa-jpl/open-source-rover/projects/1) for context

### PR DESCRIPTION
**Pull request by SConaway**
Closes #222

Refer to [this project](https://github.com/nasa-jpl/open-source-rover/projects/1) for context.

To-do:
- [x] Remove unused images (unreferenced)
- [x] Remove LaTeX files and PDFs

### CODE REVIEW

**Comment by SConaway** [comment]
Should we include a `.prettierrc` in the root dir? I'm using VSCode with the Prettier extension (a common setup) and here's mine:
```js
module.exports = {
    proseWrap: 'always',
    printWidth: 80,
};
```

**Comment by SConaway** [comment]
to figure out which images should be kept, I deleted them all, then `git restore`d any mentioned by `pandoc -s "PCB Assembly.md" -o pdf.pdf`

**Review by ericjunkins — DISMISSED** [review]
Hi @SConaway just took a look at this, I have a few comments here: 

- [x] Migrate the format to use what we have with this readme that's already done: https://github.com/nasa-jpl/open-source-rover/tree/master/mechanical/wheel_assembly . This will help solve some of the spacing issues with tables etc. 
- [ ] I think we should have the README's in their own folder so they are what appears by default on a page. So if you navigate to electrical/pcb/ it has the PCB build doc there. ([handling here](https://github.com/nasa-jpl/open-source-rover/issues/252))
- [x] We should remove the old latex documents and pdfs to remove confusion once all things are migrated

Thanks for the work on this so far, it looks like a huge amount of the work on this is done and just needs a little more to polish it

**Comment by SConaway** [comment]
@ericjunkins 

I'll get started on those changes soon, so hopefully they'll be finished in a week or so. 

I was testing in VSCode's Markdown preview, so that's probably why I didn't see the images were way too big. For the formatting changes, is there anything else I need to fix besides image sizing? Thanks!

**Comment by SConaway** [comment]
Added image size, not sure if there are any other formatting tweaks to make. Personally, I don't like the two-items-per-row in a table as it makes it hard to maintain

**Comment by SConaway** [comment]
Moved to own folder, and I don't think there are any PDFs/TeX files left for this PR.

**Comment by dcschooley** [comment]
I'll take a look at this one.

File: electrical/PCB Assembly/README.md [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+
+| **Item**               | **Ref** | **Qty** | **Image**                                                       |
+| ---------------------- | ------- | ------- | --------------------------------------------------------------- |
+| OSR Control Board      | E1      | 1       | <img src="/images/components/electronics/E1.png" height="100">  |
+| 4.7K 1/4 Watt Resistor | E7      | 1       | <img src="/images/components/electronics/E7.png" height="100">  |
+| 10K 1/4 Watt Resistor  | E8      | 4       | <img src="/images/components/electronics/E8.png" height="100">  |
+| 22K 1/4 Watt Resistor  | E9      | 4       | <img src="/images/components/electronics/E9.png" height="100">  |
+| 10K 1/2 Watt Resistor  | E10     | 1       | <img src="/images/components/electronics/E10.png" height="100"> |
+| 100nF Capacitor        | E11     | 17      | <img src="/images/components/electronics/E11.png" height="100"> |
+| Soldering Iron         | N/A     |         |                                                                 |
```
Comment (apollokit) [inline electrical/PCB Assembly/README.md:95]: Make double column

File: electrical/PCB Assembly/README.md [hunk 2]
```diff
@@ -0,0 +1,655 @@
...
+### Motor & RoboClaw Connectors
+
+| **Item**              | **Ref** | **Qty** | **Image**                                                                       |
+| --------------------- | ------- | ------- | ------------------------------------------------------------------------------- |
+| OSR Control Board     | E1      | 1       | OSR Control Board<img src="/images/components/electronics/E1.png" height="100"> |
+| 6 Pos Side Term Block | E3      | 10      | <img src="/images/components/electronics/E3.png" height="100">                  |
+| 6 Pos Top Term Block  | E4      | 5       | <img src="/images/components/electronics/E4.png" height="100">                  |
+| 10 Pos Header Socket  | E5      | 5       | <img src="/images/components/electronics/E5.png" height="100">                  |
+| 5 Pos Header Socket   | E6      | 5       | <img src="/images/components/electronics/E6.png" height="100">                  |
+| Soldering Iron        | N/A     |         |                                                                                 |
```
Comment (apollokit) [inline electrical/PCB Assembly/README.md:52]: double column

File: electrical/PCB Assembly/README.md [hunk 3]
```diff
@@ -0,0 +1,655 @@
...
+
+1. Begin by soldering the 6 Position Side entry terminal blocks **E3** into the
+   **top** side of the PCB, on the edge of the board as shown in the images
+   below. These terminal blocks will run motor power, encoder power, and encoder
+   signals between the motors/encoders and the RoboClaw motor controllers. The 6
+   terminal blocks will each be labeled with schematic reference designators
+   J17-J26 on the PCB. Be sure that the wire terminals face **OUTWARD** (away
+   from the center of the board) on all of these connectors.
+
+    ![Assembly Step 1](./../images/pcb_assembly/assembly/assembly_1.png)
```
Comment (apollokit) [inline electrical/PCB Assembly/README.md:62]: fix pics here and below (also make sure they conform to the new picture locations on master branch)

File: electrical/PCB Assembly/README.md [hunk 4]
```diff
@@ -0,0 +1,655 @@
...
+
+![Fuse and Diode Soldering](./../images/pcb_assembly/assembly/fuse.png)
+
+### USB connectors
+
+| **Item**          | **Ref** | **Qty** | **Image**                                                       |
+| ----------------- | ------- | ------- | --------------------------------------------------------------- |
+| OSR Control Board | E1      | 1       | <img src="/images/components/electronics/E1.png" height="100">  |
+| USB Connector     | E34     | 1       | <img src="/images/components/electronics/E34.png" height="100"> |
+| Diode             | E17     | 1       | <img src="/images/components/electronics/E17.png" height="100"> |
```
Comment (apollokit) [inline electrical/PCB Assembly/README.md:223]: Diode is extraneous here

File: electrical/PCB Assembly/README.md [hunk 5]
```diff
@@ -0,0 +1,655 @@
...
+
+5.  Repeat this process for all the corner motor terminal blocks, labeled
+    J23-26.
+
+## Raspberry Pi Install
+
+Next up is to verify that power to the Raspberry Pi is working. For this, you'll
+need a working operating system installed on the SD card. Take a moment now to
+follow to the
+[Software Install steps](https://github.com/nasa-jpl/open-source-rover/blob/master/Software/Software%20Steps.pdf)
```
Comment (apollokit) [inline electrical/PCB Assembly/README.md:589]: need to update this reference. Should just refer to the setup section in: https://github.com/nasa-jpl/osr-rover-code

**Review by apollokit — CHANGES_REQUESTED** [review]
looks really good! All the content is there and looks correct, just needs some formatting updates

**Comment by apollokit** [comment]
Headsup! I'm taking a crack at this right now, I think that'll minimize work overall

**Comment by SConaway** [comment]
Alright cool! Just let me know if there's anything to work on and I'll do that

**Comment by apollokit** [comment]
> Alright cool! Just let me know if there's anything to work on and I'll do that

Roger.

Also, I'll probably need you to approve my updates to the branch, I don't think I can push updates directly to your fork. Might just do a new PR. A PR for your PR. ("yo dawg, I heard you like PRs")

**Comment by SConaway** [comment]
> Also, I'll probably need you to approve my updates to the branch, I don't think I can push updates directly to your fork. Might just do a new PR. A PR for your PR. ("yo dawg, I heard you like PRs")

You should be able to, but I'm not completely sure. You can always open a PR to this branch on my fork and then I can merge it there then this one can be merged...there has to be a better way to do that. 

I think maintainers can push to PRs (if the button's checked), see the attached screenshot
CleanShot 2021-03-10 at 16 23 01

**Comment by apollokit** [comment]
> > Also, I'll probably need you to approve my updates to the branch, I don't think I can push updates directly to your fork. Might just do a new PR. A PR for your PR. ("yo dawg, I heard you like PRs")
> 
> You should be able to, but I'm not completely sure. You can always open a PR to this branch on my fork and then I can merge it there then this one can be merged...there has to be a better way to do that.
> 
> I think maintainers can push to PRs (if the button's checked), see the attached screenshot
> CleanShot 2021-03-10 at 16 23 01

Right right, I meant a PR for this branch on your fork. I'm assuming I'll have to do that because your fork's branch is the source of this PR. Shrug, we'll see.

**Comment by apollokit** [comment]
Oh, nice, it let me push straight up.

@SConaway can you give it a look-see , make sure I didn't screw anything up, and then merge?

**Review by apollokit — APPROVED** [review]

**Comment by SConaway** [comment]
@apollokit looks like you were able to add commits directly to this PR. 😃

**Comment by apollokit** [comment]
Handling @ericjunkins 's wish to reorg electrical readmes in this issue: https://github.com/nasa-jpl/open-source-rover/issues/252

**Pull request merged**